### PR TITLE
csharp/hamming

### PR DESCRIPTION
**Pull request by coding-viper**

### CODE REVIEW

File: tracks/csharp/exercises/hamming/mentoring.md [hunk 1]
```diff
@@ -82,6 +82,13 @@ public static int Distance(string strand1, string strand2)
 }
 ```
 
+### A single line method using lambda operator
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/hamming/mentoring.md:85]: I wouldn't necessarily want to encourage using a single line. Maybe rephrase this as just:

```suggestion
### A single expression
```

File: tracks/csharp/exercises/hamming/mentoring.md [hunk 2]
```diff
@@ -82,6 +82,13 @@ public static int Distance(string strand1, string strand2)
...
+
+```csharp
+public static int Distance(string first, string second) =>
+        (first.Length != second.Length) ? throw new ArgumentException() : (first.Length - Enumerable.Range(0, first.Length-1).Count(index => first[index] == second[index]));
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/hamming/mentoring.md:89]: I'd prefer a slightly different multiline formatting and using the same approach and variable names as the other examples:

```suggestion
        strand1.Length != strand2.Length
            ? throw new ArgumentException("Strands have different length")
            : strand1.Zip(strand2, (nucleotide1, nucleotide2) => nucleotide1 == nucleotide2 ? 0 : 1).Sum();
```

**Review by ErikSchierboom — COMMENTED** [review]
Thanks a lot! It is great to add a single expression example. I've left some comments.

**Review by iesoftwaredeveloper — APPROVED** [review]

**Review by iesoftwaredeveloper — DISMISSED** [review]

**Comment by angelikatyborska** [comment]
@coding-viper ping :) this PR is waiting for you to respond to @ErikSchierboom's suggestions. You can use the "commit suggestion" button below each suggestion if you agree with it.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks @coding-viper!
